### PR TITLE
round of Windows-related doc improvements

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

**Comment by SethTisue** [comment]
after-the-fact review by @adriaanm, at his leisure

File: doc/maintenance.md [hunk 1]
```diff
@@ -137,8 +137,27 @@ $ openssl req -text -noout -in scala-ci.csr
...
+You can manually bring a node online by pressing the "Launch slave
+agent" button.
+
+## Worker goes offline during manual testing?
+
+If you ssh in to a worker to poke around, that won't prevent Jenkins
+from taking the worker offline if it is otherwise idle.  This can get
+annoying.
+
+The only we way know of to force the node to stay up is, ironically,
```
Comment (adriaanm) [inline doc/maintenance.md:156]: There's a secret backdoor to disconnect the worker from Jenkins, while keeping the EC2 node up:  use "relaunch" as the message when disconnecting. https://github.com/typesafehub/ec2-start-stop/blob/master/src/main/scala/EC2StartStop.scala#L41

Comment (SethTisue) [inline doc/maintenance.md:156]: I'm not following how that's any different than the behavior of "Mark this node temporarily offline"?

Comment (adriaanm) [inline doc/maintenance.md:156]: Maybe i'm misremembering, but I think any action that takes the worker offline from jenkins's perspective would take the node down on EC2 (without the "relaunch" escape hatch)

Comment (SethTisue) [inline doc/maintenance.md:156]: Not in my experience.

**Pull request merged**